tests/models/layers/test_flash_triton_torch.py

```diff
@@ -255,7 +261,9 @@ def gen_bias(attn_impl: str):
             'rope_impl'] == 'hf'
 
         # special case that (likely) fails due to numerics
-        if clip_qkv and qk_ln and using_hf_rope and attn_type == 'grouped_query_attention':
+        if clip_qkv and (
+                qk_ln or qk_gn
+        ) and using_hf_rope and attn_type == 'grouped_query_attention':
```

Comment:
```suggestion
        if (clip_qkv and (qk_ln or qk_gn) and using_hf_rope and attn_type == 'grouped_query_attention'):
```
Wrap like this and it'll auto format better